lib/Pretty_printing.ml

```diff
@@ -171,6 +161,8 @@ and pretty_print_transformation = function
   | LowerUpper (e1, e2) ->
       "<lower=" ^ pretty_print_expression e1 ^ ", upper="
       ^ pretty_print_expression e2 ^ ">"
+  | Offset e -> "<offset=" ^ pretty_print_expression e ^ ">"
```

Comment:
[question]
Is this going to get refactored to use the `Format` library?  I'd think our pretty printing is more important for readability than our C++ code generation (for things like line breaking).

Reply:
Some day, if anyone is brave enough / feels like it. I agree it should be refactored.